### ISSUE TITLE
Should we offer to include shortcuts when there are none?

### ISSUE THREAD

**Issue by PEConn** [GoogleChromeLabs/bubblewrap#166]
Repro steps:

Run `npx @bubblewrap/cli init --manifest https://peconn.github.io/starters/manifest.json`.

Midway through you will see:

```
? Include app shortcuts?                                                                                                                                      
[] No 
```

Are we leaving that there as a hint that we support shortcuts? Or should we just remove it because the question isn't particularly useful.

**Comment by andreban** [comment]
If the users says yes, they can input shortcut details manually. But they have to type in a JSON, which is not incredibly helpful. Ideally, we'd run though a wizard so users can input multiple fields.

CC @rayankans

**Comment by rayankans** [comment]
We used to only ask if there were shortcuts available: https://github.com/GoogleChromeLabs/bubblewrap/commit/40de4c01dcb83bbd07a1b367173db84aa4c42577#diff-e0c4be7be6cd254e54c31b3e3371617bR100

Seems that that's no longer the case

**Comment by andreban** [comment]
> Are we leaving that there as a hint that we support shortcuts? Or should we just remove it because the question isn't particularly useful.

Another option is making an useful set of questions, so developers can create shortcuts even if the configuration doesn't exist in the Web Manifest.

**Comment by andreban** [comment]
This was closed via #230. We only ask when there are shortchuts.
